crates/diesel_models/src/address.rs

```diff
@@ -44,6 +45,7 @@ pub struct Address {
     pub modified_at: PrimitiveDateTime,
     pub customer_id: String,
     pub merchant_id: String,
+    pub payment_id: Option<String>,
```

Comment:
If we'll end up storing the address for each payment that includes one, won't that be including a lot of redundant data?